### PR TITLE
Docs: Use io.Seeker instead of Reset in docs

### PR DESCRIPTION
**Pull request by bloeys**
Now that https://github.com/hajimehoshi/oto/issues/172 is looking good and 19c3101d30142518729150dc41b54752776b14f8 is there we can update the docs.

This should probably be merged along with a new release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,8 +136,8 @@ func main() {
         time.Sleep(time.Millisecond)
     }
 
-    // Now that the sound finished playing, we can restart from the beginning using these two lines
-    // player.Reset()
+    // Now that the sound finished playing, we can restart from the beginning (or go to any location in the sound) using seek
+    // player.(io.Seeker).Seek(0, io.SeekStart)
```
Comment (hajimehoshi) [inline README.md:140]: Let's handle an error from Seek.

Comment (bloeys) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ func main() {
     }
 
     // Now that the sound finished playing, we can restart from the beginning (or go to any location in the sound) using seek
-    // player.(io.Seeker).Seek(0, io.SeekStart)
+    // newPos, err := player.(io.Seeker).Seek(0, io.SeekStart)
```
Comment (hajimehoshi) [inline README.md:140]:
```
if _, err := player.(io.Seeker).Seek(0, io.SeekStart); err != nil {
    return err
}
```
?

Comment (bloeys) [reply README.md:140]: ok improved it, but separate lines to make it easier for people to read.

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

**Pull request merged**